### PR TITLE
release v0.5.25

### PR DESCRIPTION
**Pull request by yondonfu**
**What does this pull request do? Explain your changes. (required)**

Version bump to v0.5.25 which supports the new L1 Rinkeby contracts for Confluence testnet.

**Specific updates (required)**

See commit history.

**How did you test each of these updates (required)**

N/A

**Does this pull request close any open issues?**

N/A

**Checklist:**

- [x] Read the [contribution guide](./doc/contributing.md)
- [x] `make` runs successfully
- [x] All tests in `./test.sh` pass
- [x] README and other documentation updated
- [x] [Pending changelog](./CHANGELOG_PENDING.md) updated

### CODE REVIEW

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -6,19 +6,12 @@
 
 ### Features ⚒
 
-- \#2157 Add support for EIP-712 typed data signing in `livepeer_cli` (@yondonfu)
-
 #### General
 
-- \#2163 Fix Session crashes right after the O sends a tx, revert #2111 Ensure `maxFeePerGas` in all transactions never exceed `-masGasPrice` (@leszko)
```
Comment (victorges) [inline CHANGELOG_PENDING.md:13]: Some of the pending changes here are not being included on the official changelog. Is that by design?

**Review by victorges — APPROVED** [review]
LGTM!

**Review by darkdarkdragon — APPROVED** [review]
LGTM

File: CHANGELOG_PENDING.md (hunk 1 above)
Comment (yondonfu) [reply CHANGELOG_PENDING.md:13]: These entries were actually already copied into the [CHANGELOG](https://github.com/livepeer/go-livepeer/blob/master/CHANGELOG.md#bug-fixes-) for the v0.5.24 release, but I just forgot to clear CHANGELOG_PENDING so those entries stuck around and now are being removed.

**Pull request merged**